app/src/androidTest/java/ch/epfl/sweng/SDP/firebase/DatabaseTest.java

```diff
@@ -45,28 +46,4 @@ public void constructBuilderWithInitialRefReturnsValidBuilder() {
         assertThat(ref.getKey(), is("tests"));
         assertThat(ref.getParent().getKey(), is("test"));
     }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void getDatabaseReferenceBuilderWithNullReference() {
-        FirebaseApp.initializeApp(InstrumentationRegistry.getContext());
-        new DatabaseReferenceBuilder(null);
-    }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void addNullChild() {
-        DatabaseReferenceBuilder databaseReferenceBuilder = init();
-        databaseReferenceBuilder.addChild(null);
-    }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void addNullChildren() {
-        DatabaseReferenceBuilder databaseReferenceBuilder = init();
-        databaseReferenceBuilder.addChildren(null);
-    }
-
-    private DatabaseReferenceBuilder init() {
-        FirebaseApp.initializeApp(InstrumentationRegistry.getContext());
-        DatabaseReference ref = Database.getReference("test.tests");
-        return new DatabaseReferenceBuilder(ref);
-    }
```

Comment:
Maybe the loss of test coverage is caused by this ?

Reply:
Yes you’re right, I removed tests that were duplicated in DatabaseReferenceBuilderUnitTest but in the process I removed this one too!

Reply:
Actually this was the init() but nevermind I'm rolling back the changes for this class